r/R/dataset.R

```diff
@@ -64,6 +73,8 @@ open_dataset <- function(sources,
                          partitioning = hive_partition(),
                          unify_schemas = NULL,
                          ...) {
+  # TODO: default to unify_schemas = TRUE when `sources` is a vector of file paths/URIs?
```

Comment:
Why would we have a different default for that case?

Reply:
The docs say
>The default is `FALSE` when creating a dataset from a file path (because there may be many files and scanning may be slow)

If the user is specifying the paths to the files, it seems likely that there would be fewer files, so maybe it would make sense to have a different default?

If you don't have any strong opinions, then we can leave it as is.

Reply:
Yeah I'd leave it, I think it would be more confusing if the default were different in those different cases.